components/proxy/src/main/resources/configuration/default.yml

```diff
@@ -87,3 +87,4 @@ url:
     unwiseCharactersToEncode:
 
 #originRestrictionCookie: yourCookieNameHere
+
```

Comment:
My problem with this change is that it results in confusing behaviour from end-user's point of view. When I'll start the server with following config, I would expect styx to open up HTTPS port *only* but it starts the insecure HTTP port too:

```
proxy:
  connectors:
    https:
      sslProvider: JDK
      certificateFile: ${STYX_HOME:classpath:}/conf/tls/testCredentials.crt
      certificateKeyFile: ${STYX_HOME:classpath:}/conf/tls/testCredentials.key
      cipherSuites:
        - TLS_RSA_WITH_AES_128_GCM_SHA256
```

The styx now started with:

```
"proxy": {
  "connectors": {
    "http": {
      "port": 8080
    },
    "https": {
      "port": 8443,
      "sslProvider": "JDK",
      "certificateFile": "/Users/mikko/src/github/styx-mikko/styx-0.7-SNAPSHOT/conf/tls/testCredentials.crt",
      "certificateKeyFile": "/Users/mikko/src/github/styx-mikko/styx-0.7-SNAPSHOT/conf/tls/testCredentials.key",
      "sessionTimeoutMillis": 300000,
      "sessionCacheSize": 20000,
      "cipherSuites": [
         "TLS_RSA_WITH_AES_128_GCM_SHA256"
      ]
    }
},```